### PR TITLE
Change how to launch Hosted UI and sample code

### PR DESCRIPTION
**Pull request by powerful23**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: js/authentication.md [hunk 1]
```diff
@@ -1244,23 +1248,33 @@ const {
...
+const { 
+    domain,  
+    redirectSignIn, 
+    redirectSignOut,
+    responseType } = config.oauth;
+
+const clientId = config.userPoolWebClientId;
+
+const url_to_google = 'https://' + domain + '/oauth2/authorize?redirect_uri=' + redirectSignIn + '&response_type=' + responseType + '&client_id=' + clientId + '&identity_provider=Google';
+const url_to_facebook = 'https://' + domain + '/oauth2/authorize?redirect_uri=' + redirectSignIn + '&response_type=' + responseType + '&client_id=' + clientId + '&identity_provider=Facebook';
```
Comment (undefobj) [inline js/authentication.md]: I think we'll be able to remove these once we have the latest `federatedSignIn` but for now I think the changes can help customers so let's add them.

File: js/authentication.md [hunk 2]
```diff
@@ -1229,11 +1229,15 @@ Amplify.configure({
 Note: An ID token is only returned if openid scope is requested. The access token can be only used against Amazon Cognito User Pools if aws.cognito.signin.user.admin scope is requested. The phone, email, and profile scopes can only be requested if openid scope is also requested.
 {: .callout .callout--info}
 
-#### Launching the Hosted UI
+#### Using Hosted UI in Web Environment
```
Comment (undefobj) [inline js/authentication.md]: Can we call this "OAuth and Hosted UI in Web Environment"?

File: js/authentication.md (hunk 1 above)
Comment (powerful23) [reply js/authentication.md]: OK

File: js/authentication.md (hunk 2 above)
Comment (powerful23) [reply js/authentication.md]: Done. Also changed "Using Hosted UI in React Native " to "OAuth and Hosted UI in React Native"

**Pull request merged**